docs/docs/admin/teamforcapella/migration.md

```diff
@@ -3,21 +3,26 @@
  ~ SPDX-License-Identifier: Apache-2.0
```

Comment:
We should also mention that the `.gitlab-ci.yml` has to be updated for diagram cache.

Reply:
Done.